docs/source/guide/deployment/planning/deployment.rst

```diff
@@ -0,0 +1,238 @@
+########################
+Deployment Configuration
+########################
+
+The Deployment Planning aims to give in-depth details to RSTUF users that will
+plan a more customized deployment.
+
+This document describes all the aspects of deploying RSTUF independent of the
+environment and focuses on the technology stacks required by RSTUF services.
+
+This document also complements our straightforward deployment guides for
+:ref:`guide/deployment/guide/docker:Docker` and
+:ref:`guide/deployment/guide/kubernetes:Kubernetes` with some detailed
+nformation.
+
+.. image:: /_static/2_2_rstuf.png
+
+Required Infrastructure Services
+################################
+
+Repository Service for TUF API and Worker some infrastructure services.
+
+* :ref:`guide/deployment/planning/deployment:Message Queue/Broker: Redis or RabbitMQ server`
+* :ref:`guide/deployment/planning/deployment:Result Backend and RSTUF Settings: Redis server`
+* :ref:`guide/deployment/planning/deployment:Database: Postgres server`
+* :ref:`guide/deployment/planning/deployment:[Optional] Content Server: Webserver`
+
+Message Queue/Broker: Redis or RabbitMQ server
+==============================================
+
+* https://redis.io
+* https://www.rabbitmq.com
+
+RSTUF API and RSTUF Worker will use the message queue/broker.
+
+RSTUF support Redis or RabbitMQ server. The decision to choose Redis or
+RabbitMQ depends on your needs.
+
+RSTUF uses Celery, and here you can find a `summary about Celery and these two
+brokers <https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html#summaries>`_.
+
+.. Caution::
+   Make sure the Redis or RabbitMQ uses a persistent storage.
+
+.. centered:: RSTUF configuration
+
+* ``RSTUF_BROKER_SERVER`` :ref:`RSTUF API <guide/repository-service-tuf-api/Docker_README:(Required) `RSTUF_BROKER_SERVER\`>`
+* ``RSTUF_BROKER_SERVER`` :ref:`RSTUF Worker <guide/repository-service-tuf-worker/Docker_README:(Required) `RSTUF_BROKER_SERVER\`>`
+
+.. Caution::
+   Both services, RSTUF API and Worker, must to access the same message
+   queue/broker.
+
+.. Note::
+   RSTUF API and Worker can use an existing Redis or RabbitMQ server in your
+   infrastructure.
+   A specific Redis database id or specific RabbitMQ vhost is recommended.
+
+   Give it a complete address to the ``RSTUF_BROKER_SERVER`` configuration on
+   RSTUF API and Worker.
+
+   - Example using Redis database id **5**: ``RSTUF_BROKER_SERVER=redis://redis/5``
+   - Example using RabbitMQ vhost **rstuf_vhost**: ``RSTUF_BROKER_SERVER="amqp://guest:guest@rabbitmq:5672/rstuv_vhost"``
+
+.. Note::
+    RSTUF requires Redis server as a
+    :ref:`result backend <guide/deployment/planning/deployment:Result Backend and RSTUF Settings: Redis server>`.
+
+    If Redis server is a good choice for you as a message queue/broker, using
+    this will simplify your deployment as it reduces one technology stack.
```

Comment:
Do you mean that having the message queue/broker is a great secondary reason for selecting Redis? Or do you mean selecting Redis over RabbitMQ reduces one technology stack if you are willing to use the Redis message queue/broker?

Reply:
I mean that selecting Redis over RabbitMQ reduces one technology stack if you are willing to use the Redis message queue/broker.